packages/web-components/fast-foundation/src/anchor/anchor.ts

```diff
@@ -94,6 +94,17 @@ export class Anchor extends FASTElement {
      * References the root element
      */
     public control: HTMLAnchorElement;
+
+    /**
+     * Manually calls the control's focus when delegatesFocus is unsupported
+     */
+    public handleUnsupportedDelegatesFocus(): boolean {
```

Comment:
I think if we call this focus, then we can probably keep from needing to add the `<template>` tag and alter the DOM structure (which would be breaking). That *should* enable us to just do the following:

```html
<fast-anchor id="anchor">Foo</fast-anchor>
```
```ts
const anchor = document.getElementById("anchor");

anchor.focus();
```